src/lsp/README.md

```diff
@@ -27,13 +18,13 @@ You need a recent enough VSCode (>= 1.31)
 
 ## Using with Emacs
 
-`lp-lsp` will work out of the box with emacs and eglot, to do so,
-install [eglot](https://github.com/joaotavora/eglot) [using `M-x
-package-install eglot RET` should work], then do `M-x eglot` and
-specify as a server `lp-lsp --std`.
+Lambdapi LSP server will work out of the box with emacs and eglot,
+to do so, install [eglot](https://github.com/joaotavora/eglot)
+[using `M-x package-install eglot RET` should work], then do
+`M-x eglot` and specify as a server `lambdapi --lsp-server --std`.
```

Comment:
as server?

Reply:
Why --std?

Reply:
I did not write this doc, but indeed, `--std` should be `--standard-lsp`. This is fixed.